Makefile

```diff
@@ -126,6 +126,10 @@ metrics:
 extended_metrics:
 	cd metrics && $(PERL) ./mapping.pl | $(PYTHON) ./extract_tfms.py | $(PYTHON) ./extract_ttfs.py | $(PYTHON) ./format_json.py --width > ../src/fontMetricsData.js
 
+unicode:
+	cd src && $(NODE) unicodeMake.js >unicodeSymbols.js
+src/unicodeSymbols.js: unicode
```

Comment:
I wonder if we can run this as part of the build and then not even bother with committing the result.  That we don't have to worry about whether people are running it when they should or not.

Reply:
My motivation for committing it is so that people wouldn't worry about when to run it.  (You need to run it whenever we add support for a new accent, or when the Unicode spec changes, neither of which I imagine being very common.)  Another advantage is that we can directly see what characters are supported, so it's easy to complain about a missing one, and for PRs to show what additional characters are being supported.

If we do it during build, would we also have to do it when running the test server...?  This approach felt more complicated to me, but I'm happy to have a specific suggestion.

Reply:
Let's leave it like this for now.  I definitely like it committing it from a documentation point of view.  I don't think we have much work left to do on accents so there shouldn't be too many PRs which is affected by this.